.github/workflows/release.yml

```diff
@@ -0,0 +1,32 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "**/charts/*"
+      - ".github/workflows/release.yml"
+      - ".releaserc.yaml"
```

Comment:
Wouldn't you want this to trigger when any of the source files change?

Reply:
yes, would a new path of "**.js" be sufficient? where it would catch changes to all js files that are changed?

Reply:
I suppose it depends - if there are tests in this repository, then that would probably match too much.  For example, we don't want to cut a release if someone submitted a PR to refactor a test.

That is probably fine for now.

Reply:
that makes sense, i've changed the paths to be more specific